CryptoAnalysis/src/main/java/crypto/reporting/ReporterHelper.java

```diff
@@ -69,13 +69,13 @@ public static String generateReport(List<CrySLRule> rules, Collection<IAnalysisS
 			}
 			report += "\n";
 		}
-		report += "======================= CogniCrypt Summary ==========================\n";
+		report += "======================= CryptoAnalysis Summary ==========================\n";
 		report += String.format("\tNumber of CrySL rules: %s\n", rules.size());
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [FS: Format string should use %n rather than n (VA_FORMAT_STRING_USES_NEWLINE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#fs-format-string-should-use-n-rather-than-n-va-format-string-uses-newline).
This format string includes a newline character (\n). In format strings, it is generally  preferable to use %n, which will produce the platform-specific line separator.